src/utils/utils.coffee

```diff
@@ -72,14 +72,19 @@ Ember.Table.RowArrayController = Ember.ArrayController.extend
     subControllers = @get '_subControllers'
     subController = subControllers[idx]
 
+    @set('lastItem', subController) if @isLastItem(idx)
     return subController if subController
     subController = @get('itemController').create
       target: this
       parentController: @get('parentController') or this
       content: object
     subControllers[idx] = subController;
+    @set('lastItem',subController) if @isLastItem(idx)
```

Comment:
it would be great if we could restructure this logic such that `@set('lastItem',subController) if @isLastItem(idx)` only needs to be called once. Something like:
```
unless subController
  subController = @get('itemController').create
    target: this
    parentController: @get('parentController') or this
    content: object
  subControllers[idx] = subController;
@set('lastItem',subController) if @isLastItem(idx)
```
There should also be a comment explaining why we need to do this.